<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumberPolinoms</class>
 <widget class="QWidget" name="NumberPolinoms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backPolinoms">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>791</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Полиномы</string>
   </property>
   <widget class="QPushButton" name="firstPolinom">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ввести первый полином</string>
    </property>
   </widget>
   <widget class="QPushButton" name="secondPolinom">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ввести второй полином</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>k(при x^k):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditK">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Рациональное число:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="polinomsRatioN">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="polinomsRatioD">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="push_ADD_PP_P">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ADD_PP_P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_SUB_PP_P">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SUB_PP_P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_MUL_PQ_P">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MUL_PQ_P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_MUL_Pxk_P">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MUL_Pxk_P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_DEG_P_N">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DEG_P_N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_LED_P_Q">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LED_P_Q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_GCF_PP_P">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GCF_PP_P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_FAC_P_Q">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FAC_P_Q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_DIV_PP_P">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DIV_PP_P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_DER_P_P">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DER_P_P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_MOD_PP_P">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MOD_PP_P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_MUL_PP_P">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MUL_PP_P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_NMR_P_P">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NMR_P_P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="faqP">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>10</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>791</width>
     <height>121</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>789</width>
      <height>119</height>
     </rect>
    </property>
    <widget class="QTextEdit" name="answerText">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>791</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ответ:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
